core/src/main/java/io/grpc/internal/BinaryLogProvider.java

```diff
@@ -61,11 +64,32 @@ public static BinaryLogProvider provider() {
   /**
    * Creates a new {@link MethodDescriptor} where both marshallers are {@code InputStream} based.
    */
-  static <ReqT, RespT> MethodDescriptor<InputStream, InputStream> wrapMethod(
-      MethodDescriptor<ReqT, RespT> method) {
+  private static MethodDescriptor<InputStream, InputStream> wrapMethod(
+      MethodDescriptor<?, ?> method) {
     return method.toBuilder(IDENTITY_MARSHALLER, IDENTITY_MARSHALLER).build();
   }
 
+  /**
+   * Wraps a {@link ServerMethodDefinition} such that it performs binary logging if needed.
+   */
+  final <ReqT, RespT> ServerMethodDefinition<?, ?> wrapMethodDefinition(
```

Comment:
These ReqT, RespT look unused. Use `<?, ?>` instead?

Reply:
This calls a helper function that uses it later on:
https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/ServerInterceptors.java#L243

Reply:
But it doesn't seem like they are necessary here.